src/reflect/scala/reflect/internal/SymbolTable.scala

```diff
@@ -363,11 +363,17 @@ abstract class SymbolTable extends macros.Universe
 
     // Weak references so the garbage collector will take care of
     // letting us know when a cache is really out of commission.
-    import java.lang.ref.WeakReference
+    import java.lang.ref.{WeakReference, Reference}
```

Comment:
unused import?

Reply:
fixed in https://github.com/scala/scala/pull/6223/commits/abef11199b9e140e7241fa17b3f5e08ea117c453